src/coreclr/vm/tieredcompilation.cpp

```diff
@@ -1105,9 +1102,7 @@ CORJIT_FLAGS TieredCompilationManager::GetJitFlags(PrepareCodeConfig *config)
 
         methodDesc->GetLoaderAllocator()->GetCallCountingManager()->DisableCallCounting(nativeCodeVersion);
         nativeCodeVersion.SetOptimizationTier(NativeCodeVersion::OptimizationTierOptimized);
-    #ifdef FEATURE_INTERPRETER
         flags.Set(CORJIT_FLAGS::CORJIT_FLAG_MAKEFINALCODE);
```

Comment:
The flag was previously set only for interpreter here and in the code below. Are these changes intentional?

Reply:
Fixed - delete the flag. (I went back and forth between keeping the flag for future proofing vs. deleting it and ended up with mixed edits.)